.gitignore

```diff
@@ -3,4 +3,12 @@ bazel-*
 node_modules/
 .pnpm-*
 .DS_Store
-.bazelrc.user
\ No newline at end of file
+.bazelrc.user
+
+# Disable lockfile for now. It is unstable.
+# https://github.com/bazelbuild/bazel/issues/19026
+# https://github.com/bazelbuild/bazel/issues/19621
+# https://github.com/bazelbuild/bazel/issues/19971
+# https://github.com/bazelbuild/bazel/issues/20272
+# https://github.com/bazelbuild/bazel/issues/20369
+MODULE.bazel.lock
```

Comment:
nit: this is redundant right? I'd never have this file appear since it's not produced.

I think it's better not to ignore it, because when we do decide to create one, it should get checked in.

Reply:
Yes. Tho its a defence in depth since someone will inevitably add a new e2e test and forget to disable the lock file in that .bazelrc without noticing. OTOH, with this here, when we go to check in the files finally it will be obvious it is .gitignored.